### ISSUE TITLE
Where is the documentation

### ISSUE THREAD

**Issue by PilotJack** [linkedin/Burrow#386]
I've been working with Borrow for some weeks now. Getting it running was a challenge because the documentation is sparse at best.  I'd like to know where to find certain information like:

- How does the windowing really work, and how do I control it?  What is the sampling interval? 
- Where is configuration explained in more detail than in the Wiki?  Current docs list the
   configuration parameters, but little is available in terms of working examples with 
   coherent explanations. Basically to get something going currently involves a lot of guesswork.
- In short, where is a real writeup explaining this thing at a level of detail where we can work 
   with it without a lot of guesswork?

Thanks,

Jack

**Comment by opsj** [comment]
+1

**Comment by PilotJack** [comment]
Thanks for the reply. In particular I need to know how windowing works.  Everything I've been able to find on this is too vague to really understand how often samples happen, how many samples are in a window, how long the window is, what the interval is between commits, even what the definition of "commit" is. This is very fundamental stuff that every user should have access to.  Thanks again!
